app/deliver_grant_funding/templates/deliver_grant_funding/grant_setup/ggis_number.html

```diff
@@ -0,0 +1,83 @@
+{% extends "common/base.html" %}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+{% from "govuk_frontend_jinja/components/error-summary/macro.html" import govukErrorSummary %}
+
+{% block pageTitle %}
+  Do you have a GGIS reference number? - Add a grant - MHCLG Funding Service
+{% endblock pageTitle %}
+
+{% set active_item_identifier = "grants" %}
+
+{% block beforeContent %}
+  {{
+    govukBackLink({
+        "text": "Back",
+        "href": url_for("deliver_grant_funding.grant_setup_intro")
+    })
+  }}
+{% endblock beforeContent %}
+
+{% set before_inputs_text %}
+  <p class="govuk-body govuk-!-margin-top-7">
```

Comment:
Let's please remove the custom margins here - trust the stuff that gets spit out by GOV.UK Frontend and govuk-frontend-jinja/wtf natively unless there is a really clear directive that we need to tweak padding/margins/layout for some reason. If that's the case, let's probe that a little more and think about how we can make sure it's being applied consistently in the future.